Tests/SourceKitLSPTests/SourceKitTests.swift

```diff
@@ -10,10 +10,12 @@
 //
 //===----------------------------------------------------------------------===//
 
+import ISDBTestSupport
```

Comment:
Why do we need to import `ISDBTestSupport` here?

Reply:
For `TestLocation`.

Reply:
👍 Makes sense